packages/apps-v5/src/commands/ps/type.js

```diff
@@ -3,15 +3,15 @@
 let cli = require('heroku-cli-util')
 const { sortBy, compact } = require('lodash')
 
-const costs = { 'Free': 0, 'Eco': 0, 'Hobby': 7, 'Standard-1X': 25, 'Standard-2X': 50, 'Performance-M': 250, 'Performance': 500, 'Performance-L': 500, '1X': 36, '2X': 72, 'PX': 576 }
+const costs = { 'Free': 0, 'Eco': 0, 'Basic': 7, 'Standard-1X': 25, 'Standard-2X': 50, 'Performance-M': 250, 'Performance': 500, 'Performance-L': 500, '1X': 36, '2X': 72, 'PX': 576 }
```

Comment:
Let's leave `Hobby` in here too so that we don't have any concern about timing with releasing this.

Reply:
@tevanoff 👍